R/auth.R

```diff
@@ -29,15 +29,18 @@ deleteSessionInfo <- function () {
 #' Authenticate with the Crunch API
 #'
 #' Note that you can store your Crunch account info in your .Rprofile under
-#' "crunch.email" and "crunch.pw" for convenience. If you do so, you can simply
-#' \code{login()} to authenticate. For running batch jobs, this could be
+#' `crunch.email` and `crunch.pw` for convenience. If you do so, you can simply
+#' `login()` to authenticate. For running batch jobs, this could be
 #' particularly useful. However, be warned that storing your
 #' password in a plain text file such as .Rprofile is a security risk (though
 #' perhaps less so than in every .R script you write), and we
 #' cannot officially recommend that you do so.
 #'
+#' Additionally, your email and password can be stored in and read from the 
+#' environmental variables `R_CRUNCH_EMAIL` and `R_CRUNCH_PW` respectively.
```

Comment:
Should update the README to reflect how you can specify credentials in env vars too now.